src/abstractdataframe/show.jl

```diff
@@ -458,41 +485,64 @@ function showrows(io::IO,
     return
 end
 
-#' @exported
-#' @description
-#'
-#' Render an AbstractDataFrame to an IO system. The specific visual
-#' representation chosen depends on the width of the REPL window
-#' from which the call to `show` derives. The dynamic response 
-#' to screen width can be configured using the `allcols` argument.
-#'
-#' @param io::IO The IO system to which `df` will be printed.
-#' @param df::AbstractDataFrame An AbstractDataFrame.
-#' @param allcols::Bool Should only a subset of columns that fits
-#'        the device width be printed? Defaults to `false`.
-#' @param rowlabel::Symbol What label should be printed when rendering the
-#'        numeric ID's of each row? Defaults to `"Row"`.
-#' @param displaysummary::Bool Should a brief string summary of the
-#'        AbstractDataFrame be rendered to the IO system before printing the
-#'        contents of the renderable rows? Defaults to `true`.
-#'
-#' @returns `nothing` value.
-#'
-#' @examples
-#'
-#' df = DataFrame(A = 1:3, B = ["x", "y", "z"])
-#' show(stdout, df, false, :Row, true)
+"""
+    show([io::IO,] df::AbstractDataFrame;
+         allrows::Bool = !get(io, :limit, false),
+         allcols::Bool = !get(io, :limit, false),
+         allgroups::Bool = !get(io, :limit, false),
+         splitcols::Bool = get(io, :limit, false),
```

Comment:
Looks good. The small comments to the documentation are:
* is here that you have named it `split` not `splitcols` in the implementation an later in the documentation (actually I like `splitcols` better).
* there is a tricky part that defaults are different when you pass `io` than when you do not pass it. Maybe simply add a comment at the end of the docs that with default `io` which is `stdout` we have different defaults.

Reply:
Woops. I actually wanted to ask your opinion about the name of the argument. Let's go with `splitcols` then.

I've also added a mention of the special defaults when `io` is omitted. It doesn't sound ideal that we have to do this, but that's a deeper issue with printing in Julia.